engine/script/src/script.cpp

```diff
@@ -207,7 +206,6 @@ namespace dmScript
         context->m_ContextTableRef = Ref(L, LUA_REGISTRYINDEX);
 
         InitializeTimer(context);
-        InitializeExtensions(context);
 
         for (HScriptExtension* l = context->m_ScriptExtensions.Begin(); l != context->m_ScriptExtensions.End(); ++l)
```

Comment:
Previously, each native extension, was part of this "script extension".
However, as not all extensions are adding script functionality, it also seemed a bit convoluted to bake them together:
Also, the engine "app initializes" the extensions, and the script "initializes" the extensions.
It just felt strange in terms of ownership and life times.